src/System.Xml.XmlSerializer/src/System/Xml/Serialization/XmlAnyElementAttributes.cs

```diff
@@ -16,7 +17,7 @@ namespace System.Xml.Serialization
     /// </devdoc>
     public class XmlAnyElementAttributes : IList
     {
-        private ArrayList _list;
+        private List<object> _list;
```

Comment:
@justinvp He just changed `MemberMappingComparer` to implement the generic interface [here](https://github.com/dotnet/corefx/pull/7765/files#diff-6ffe8b01d9061d87dac3108040ae3d2eR796); as long as it isn't exposed in the contract (which it [isn't](https://github.com/dotnet/corefx/blob/master/src/System.Xml.XmlSerializer/ref/System.Xml.XmlSerializer.cs#L26)), it shouldn't matter, right? (Also the `IList`-based methods that take an `object` don't appear to be exposed, either.)